_sub/monitoring/helm-exporter/dependencies.tf

```diff
@@ -61,7 +61,38 @@ locals {
           "replicas" = var.replicas
         }
         "intervalDuration" = var.interval_duration
+        "namespaces"       = var.target_namespaces
+        "config" = {
+          "helmRegistries" = {
+            "override" = [
+              {
+                "registry" = {
+                  "url" = "https://helm.traefik.io/traefik/index.yaml"
+                },
+                "charts" = [
+                  "traefik"
+                ]
+              },
+              {
+                "registry" = {
+                  "url" = "https://kubernetes-sigs.github.io/metrics-server/index.yaml"
+                },
+                "charts" = [
+                  "metrics-server"
+                ]
+              },
+              {
+                "registry" = {
+                  "url" = "https://prometheus-community.github.io/helm-charts/index.yaml"
+                },
+                "charts" = [
+                  "kube-prometheus-stack"
+                ]
+              }
+            ]
+          }
+        }
```

Comment:
This block seems like it should be configurable via variable also.

Reply:
what variable structure are you proposing here? seems a bit complicated for a variable..

Reply:
list of objects with registry and list of charts

Reply:
something like this you mean ? 
```
variable "helm_exporter_target_charts" {
  type = list(object({
    registry  = string
    charts    = list(string)
  }))
}
```

Reply:
Yes, exactly.